README.md

```diff
@@ -1,2 +1,101 @@
-# keycloak-operator
-A Kubernetes Operator based on the Operator SDK for syncing resources in Keycloak
+[![Build Status](https://travis-ci.org/keycloak/keycloak-operator.svg?branch=master)](https://travis-ci.org/keycloak/keycloak-operator)
+[![Go Report Card](https://goreportcard.com/badge/github.com/keycloak/keycloak-operator)](https://goreportcard.com/report/github.com/keycloak/keycloak-operator)
+[![Coverage Status](https://coveralls.io/repos/github/keycloak/keycloak-operator/badge.svg?branch=master)](https://coveralls.io/github/keycloak/keycloak-operator?branch=master)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+# Keycloak Operator
+A Kubernetes Operator based on the Operator SDK for syncing resources in Keycloak.
+
+## Current status
+Currently in development. Will eventually replace https://github.com/integr8ly/keycloak-operator 
+
+## Supported Custom Resources
+| *CustomResourceDefinition*                        | *Description*                                            |
+| ------------------------------------------------- | -------------------------------------------------------- |
+| [Keycloak](./deploy/crds/Keycloak.yaml)           | Manages, installs and configures Keycloak on the cluster |
+| [KeycloakRealm](./deploy/crds/KeycloakRealm.yaml) | Represents a realm in a keycloak server                  |
+
+## Local Development
+*Note*: You will need a running Kubernetes or OpenShift cluster to use the Operator
+
+1.  clone this repo to `$GOPATH/src/github.com/keycloak/keycloak-operator`
+2.  run `make setup/mod cluster/prepare`
+3.  run `code/run`
+-- The above step will launch the operator on the local machine
+-- To see how do debug the operator or how to deploy to a cluster, see below alternatives to step 3
+4. In a new terminal run `make cluster/create/examples`
+
+To clean the cluster (Removes CRDs, CRs, RBAC and namespace)
+1. run `make cluster/clean`
+
+### Alternative Step 3: Debug in Goland
+Debug the operator in [Goland](https://www.jetbrains.com/go/)
+1. go get -u github.com/go-delve/delve/cmd/dlv
+2. Create new `Go Build` debug configuration
+3. Change the properties to the following
+```
+* Name = Keycloak Operator
+* Run Kind = File
+* Files = <project full path>/cmd/manager/main.go
+* Working Directory = <project full path>
+* Environment = KUBERNETES_CONFIG=<kube config path>;WATCH_NAMESPACE=keycloak
+```
+3. Apply and click Debug Keycloak operator
+
+### Alternative Step 3: Debug in VSCode
+Debug the operator in [VS Code](https://www.jetbrains.com/go/)
```

Comment:
I guess you mistook the link here (should be vscode, not goland)

Reply:
Excellent spotting and thank you for the review. This is now fixed.